src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,15 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
+    let startFrom = fromIndex;
```

Comment:
загалом ця змінна не потрібна.
```suggestion
    const startFrom = fromIndex;
```